src/eval.jl

```diff
@@ -54,7 +66,7 @@ handle("eval") do data
   lock(evallock)
   result = @dynamic let Media.input = Editor()
     withpath(path) do
-      @errs include_string(mod, text, path, line)
+      @errs include_string(mod, text, path, line) ends_with_semicolon(text)
```

Comment:
> regarding evaluating multi-line in the editor, it works just as before but suppresses the output if the last line ends with semi-colon

That's not quite true actually. For example:

```julia
Base.REPL.ends_with_semicolon("""
  function foo()
    foo; # bar
  end
  """) # -> true
```

So the output of that code block would be suppressed, which could be confusing.

However, I think this is sufficiently unlikely to cause problems in practice that we can just wait and fix it if it does.